code/modules/antagonists/wizard/abilities/animal.dm

```diff
@@ -115,14 +115,18 @@ var/list/animal_spell_critter_paths = list(/mob/living/critter/small_animal/cat,
 		smoke.attach(target)
 		smoke.start()
 
-		if (target.mind && (target.mind.assigned_role != "Animal") || (!target.mind || !target.client))
+		if (target.mind || (!target.mind || !target.client))
```

Comment:
```suggestion
		if (!target.mind || !target.client)
```
Currently checking mind || !mind which always returns TRUE, think the original mind check was just to check for animal?